iTunesUDataPurge/RemoveITunesUCourses.java

```diff
@@ -129,6 +133,15 @@ static public String getCredentialToken(String displayName, String emailAddress,
 	
 	static public void removeCourses(HashMap<String, String> siteIds, String prefix, String destination, String displayName, String emailAddress, String username, String userIdentifier)
```

Comment:
So here is the definition of removeCourses. However, I did not fine it is called anywhere inside this patch.